themes/edge.edx.org/lms/static/sass/partials/base/_certificates.scss

```diff
@@ -0,0 +1,27 @@
+// Certificate overrides for edge.edx.org
+
+.certificate-card {
+    &.mode-honor, &.mode-audit {
```

Comment:
Curious, can this really be audit?  Seems confusing.  Does it deserve a comment?

Reply:
It probably deserves a comment. edx.org no longer supports audit certificates, but there are legacy certificates that might be rendered. @marcotuts suggested making them render the same as honor.